src/libopensc/cardctl.h

```diff
@@ -962,20 +980,24 @@ typedef struct sc_cardctl_piv_genkey_info_st {
 #define SC_OPENPGP_KEYFORMAT_RSA_CRT	2
 #define SC_OPENPGP_KEYFORMAT_RSA_CRTN	3
 
+#define SC_OPENPGP_KEYFORMAT_EC_STD	0
+#define SC_OPENPGP_KEYFORMAT_EC_STDPUB	0xFF
```

Comment:
clang-format suggested the following change:


```
-#define SC_OPENPGP_KEYFORMAT_EC_STD	0
-#define SC_OPENPGP_KEYFORMAT_EC_STDPUB	0xFF
+#define SC_OPENPGP_KEYFORMAT_EC_STD    0
+#define SC_OPENPGP_KEYFORMAT_EC_STDPUB 0xFF
```

<!-- identifier: clang-format -->
<!-- File: src/libopensc/cardctl.h -->